CHANGELOG.md

```diff
@@ -1,6 +1,34 @@
 Change Log
 ==========
 
+Version 2.0.0 *(15 February 2024)*
+-------------------------------------------
+**What's new**
+* **[Android Platform]**
+  * Supports [CleverTap Android SDK v6.0.0](https://github.com/CleverTap/clevertap-android-sdk/blob/master/docs/CTCORECHANGELOG.md#version-600-january-15-2024).
+
+
+* **[iOS Platform]**
```

Comment:
I believe the iOS PR has resolved the changeLog todos.

Reply:
Yess. They have been resolved